Source/ConfigCatWrappers/Private/ConfigCatEvaluationWrapper.cpp

```diff
@@ -0,0 +1,117 @@
+﻿// Fill out your copyright notice in the Description page of Project Settings.
```

Comment:
What is the standard? Is this line needed here?

Reply:
that is an oversight on my side. I have adjusted it with the proper copyright.